### PR TITLE
@orta => 3.4 fixes

### PR DESCRIPTION
**Pull request by ashfurrow**
Christine asked for a 3.4 release for Monday, fixing these things. 

No tests yet. Here's what I want to do:

- [x] Write code for fixes
- [x] Orta reviews code 
- [x] I write tests(ish)

After Orta review's the code, we at least have a build to test for the auction.

### CODE REVIEW

**Comment by orta** [comment]
Looks good.

**Comment by ashfurrow** [comment]
OK, so checks are in. Once merged, I'll make a build and get @alessandra-castillo's help testing.

File: Kiosk/App/Models/Artwork.swift [hunk 1]
```diff
@@ -60,7 +60,11 @@ public class Artwork: JSONAble {
         }
 
         if let imageDicts = json["images"].object as? Array<Dictionary<String, AnyObject>> {
-            artwork.images = imageDicts.map({ return Image.fromJSON($0) as Image })
+            // There's a possibility that image_versions comes back as null from the API, which fromJSON() is alergic to.
```
Comment (orta) [inline Kiosk/App/Models/Artwork.swift]: allergic

**Comment by orta** [comment]
Ready to merge :+1:

**Pull request merged**